### DISCUSSION TITLE
Session ended right after the link processed.

### DISCUSSION THREAD

**Discussion by venom7aries** [jaykali/maskphish#60] [Q&A] [answered]
The session ended right after i generated the phishing URL. I totally like this repository but whats the point of call it a phishing link if the session ended right after it? It should show the IP and the details we give in. Or atleast show the IP? This is pretty much useless at this point. We made a custom phishing url, and thats it, it ended right there. If this is a bug, please fix it asap, or please work on this one. Thanks

**Comment by jaykali** [comment]
Can you please show a screenshot?

**Comment by jaykali** [comment]
Maskphish can't end the sessions. It just trick the URL shown. But forward everyone to the given URL.

**Comment by venom7aries** [comment]
But phishing links are to be logged in the same session right? How am I
supposed to see the victim’s IP and the details if the session is literally
over right after processing the link?

**Comment by jaykali** [comment, marked as answer]
Let me explain what Maskphish do.

1. It collects the URL of phishing link from user.
2. Then Collect the Masking URL (Normal websites like Facebook Github etc).
3. Then collect some social engineering words to make the things more realistic.
4. Then it short the phishing URL using URL shortner services to give a proper look.
5. Then add everything in a way `maskingurl-social-engineering-words@phishingURLshort` That's it. Like if you open this url on browser [https://facebook.com-i-needmoney@youtube.com](https://facebook.com-i-needmoney@youtube.com) it will redirect you to Youtube Not in Facebook. Because browser looks after @.

You can see everything is so simple on Maskphish. It just tricks the URL. Nothing else. You can read the codes. It's open-source.

**Reply by rolexzad** [reply]
How can i get the code or the software to do that

**Comment by venom7aries** [comment]
Damn now i get it. Dude fr thanks for letting me know.
